lib/pf/services/manager/collectd.pm

```diff
@@ -58,7 +61,10 @@ sub generateCollectd {
     $tags{'graphite_host'} = "$Config{'monitoring'}{'graphite_host'}";
     $tags{'graphite_port'} = "$Config{'monitoring'}{'graphite_port'}";
     $tags{'hostname'}      = hostname;
-    $tags{'db_host'}       = "$Config{'database'}{'host'}";
+    $tags{'db_host'}
+        = $cluster_enabled
+        ? $management_network->tag('ip')
```

Comment:
Not sure we can expect a MySQL database on this in cluster.

Unless its used for something else than to connect to MySQL